### DISCUSSION TITLE
v20 installing: maybe offer Europe locale

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#3893] [Ideas]
You have this now also in windows. Often you get the advice to use Ireland, but than your decimal separator is different. Best would be to check a bit how this is done in windows.

**Comment by luigifab** [comment]
Do you have an example?

I know that LibeOffice (and OpenOffice and StarOffice) is stupid because it don't understand decimal separator when it differ from locale (for example point instead of comma), but for OpenMage...
